src/faq.hbs

```diff
@@ -0,0 +1,94 @@
+				<div class="tab-content">
+					Ja, du kannst die Urlaubsverwaltung 30 Tage lang unverbindlich und kostenlos testen.
+					Während der Testphase sind alle Funktionen der Software vollständig freigeschaltet.
+				</div>
+			</div>
+			<div class="tab">
+				<input type="checkbox" id="rd2" name="rd">
+				<label class="tab-label" for="rd2">Was passiert nach der kostenlosen Testphase?</label>
+				<div class="tab-content">
+					Wenn du über die kostenlose Testphase hinaus weiter Interesse an der Urlaubsverwaltung hast,
+					bestätige die Teilnahme per E-Mail an <a href="mailto:info@urlaubsverwaltung.cloud?subject=Teilnahme%20am%20kostenpflichtigen%20Service&body=Liebes%20Urlaubsverwaltung-Team%0D%0A%0D%0AIch%20bestätige%20die%20Teilnahme%20am%20kostenpflichtigen%20Service%20und%20stimme%20damit%20den%20geltenden%20AGB%20und%20Datenschutzbestimmungen%20zu.%0D%0A%0D%0AViele%20Grüße">
+					info@urlaubsverwaltung.cloud</a>. Solltest du kein Interesse mehr haben, löschen wir deine Urlaubsverwaltung mit allen darin generierten Daten nach der Testphase.
+				</div>
+			</div>
+			<div class="tab">
+				<input type="checkbox" id="rd3" name="rd">
+				<label class="tab-label" for="rd3">Wie kann ich an der kostenlosen Testphase teilnehmen?</label>
+				<div class="tab-content">
+					Melde dich <a href="https://registry.apps.urlaubsverwaltung.cloud/registration">hier</a> kostenlos an und registriere dir damit deine eigene Urlaubsverwaltung.
+					Sie steht dir in wenigen Minuten zur Verfügung. Ohne Terminabsprache oder langen Wartezeiten.
+				</div>
+			</div>
+			<div class="tab">
+				<input type="checkbox" id="rd4" name="rd">
+				<label class="tab-label" for="rd4">Welche Zahlungsarten gibt es und welche Kündigungsfristen gelten?</label>
+				<div class="tab-content">
+					Du bekommst monatlich nach dem Leistungszeitraum eine Rechnung per E-Mail zugestellt.
+					Kündigen kannst du mit 14 Tagen zum jeweiligen Ende eines jeden Vertragsmonats.
+				</div>
+			</div>
+			<div class="tab">
+				<input type="checkbox" id="rd5" name="rd">
+				<label class="tab-label" for="rd5">Wieviel kostet mich die Urlaubsverwaltung?</label>
+				<div class="tab-content">
+					Jeder registrierte Benutzer der den Status aktiv hat, kostet 1,50 € (zzgl.MwSt) pro Monat. Kommt ein neuer
+					Benutzer während des Leistungszeitraumes hinzu, wird er für diesen Zeitraum nur anteilig abgerechnet.
+					Wird ein Benutzer auf den Status inaktiv gesetzt, findet er keine Beachtung in der Rechnung ab dem darauf folgenden Leistungszeitraum.
+				</div>
+			</div>
+			<div class="tab">
+				<input type="checkbox" id="rd6" name="rd">
+				<label class="tab-label" for="rd6">Gibt es eine Kalenderintegration?</label>
+				<div class="tab-content">
+					Jeder Benutzer der Urlaubsverwaltung kann seinen eigenen Kalender, sowie einen Kalender pro Abteilung abonnieren.
+					Ist der Firmen-weite Kalender freigeschaltet, lässt sich auch dieser von jedem Mitarbeiter abonnieren.
```

Comment:
Eventuell könnte man noch in einem Satz erklären werden Firmenweiten Kalender freischaltet?

Reply:
Fehlt in deiner Frage das Wort "wer"? FAQ Antworten sollten kurz und prägnant sein. Wer spielt an der Stelle keine Rolle, nur dass es grundsätzlich möglich ist.